cookiecutter.json

```diff
@@ -1,10 +1,13 @@
 {
-    "author_name": "author",
-    "author_email": "author@example.com",
-    "project_name": "my_project",
-    "project_title_line_1": "My LaTex Project",
+    "author_name": "author name",
+    "author_email": "author@domain.com",
+    "project_name": "new_project",
+    "project_title_line_1": "a new project",
     "project_title_line_2": "",
     "project_title_line_3": "",
+    "initial_git_commit": "y",
+    "enter_git_origin": "",
+    "push_initial_commit": "y",
```

Comment:
Rather than asking these values at the start, they should be asked only when the directory is successfully created.

That is only when I want to see if I want to commit and push it.

Reply:
Do you mean, to put this functionality in `post_gen_project.sh`?

Reply:
probably yes